agent/util-scripts/test-bin/rpmdev-vercmp

```diff
@@ -3,18 +3,27 @@
 args="${@}"
 echo "${0} ${args}" >> ${_testlog}
 
+# We are mocking the behavior of rpmdev-vercmp where the behavior of this
+# mock is directed by the 2nd arguments name.  This allows specific test
+# cases to request a certain behavior from rpmdev-vercmp.
+
 if [[ "${2}" == "not-greater-equal-1.0.0" ]]; then
+    echo "${1} <= ${2}"
     _rc=12
 elif [[ "${2}" == "greater-equal-1.0.0" || "${2}" == "greater-1.0.0" ]]; then
+    echo "${1} >= ${2}"
     _rc=11
 elif [[ "${2}" == "not-greater-1.0.0" ]]; then
+    echo "${1} == ${2}"
     _rc=0
```

Comment:
If the caller's expectation is that the comparison should report "_not_ greater than", shouldn't the echo'd string be `<=` rather than `==`?

The return value of `0` indicates that the version are _equal_...which is not a proper match with "not greater than"....

Reply:
I'll do a better job of documenting the expected behavior, but `rpmdev-vercmp` only emits `==` (0), `<` (11), `>` (12), on string comparisons:

```
[pportant@gandalf util-scripts]$ rpmdev-vercmp fio-3.21 fio-3.22
fio-3.21 < fio-3.22
[pportant@gandalf util-scripts]$ rpmdev-vercmp fio-3.22 fio-3.22
fio-3.22 == fio-3.22
[pportant@gandalf util-scripts]$ rpmdev-vercmp fio-3.23 fio-3.22
fio-3.23 > fio-3.22
[pportant@gandalf util-scripts]$
```

To mimic the behavior we really should only return `==` (0) or `<` (12), so I chose `==` (0).